tests/test_interventionFunctions.py

```diff
@@ -0,0 +1,53 @@
+from SOARback import interventionFunctions
+import sqlite3
+import json
+
+db = 'SOARback/db/SOARback_interventions.db'
+test_data = [2, 'alice', '2020-12-18 12:12:12', 'left', 3, 0]
+
+def connection_start(table):
+    conn = interventionFunctions.connect()
+    conn.execute(f"DELETE FROM {table}")
+    conn.commit()
+    conn.close()
+
+def connection_end(conn, table):
+    conn.execute(f"DELETE FROM {table}")
+    conn.commit()
+    conn.close()
+
+
+def test_addEntry():
```

Comment:
A common thing I've done in the past for tests is to have the test name be something like "Class/Function_WhatTestIsDoing_ExpectedResult". Not sure if we want to adopt something like that...

Reply:
I think that would be a good idea and lead to more robust tests as well